Corona-Warn-App/src/main/res/navigation/nav_graph.xml

```diff
@@ -503,8 +503,8 @@
     <!-- New Release -->
     <fragment
         android:id="@+id/newReleaseInfoFragment"
-        android:name="de.rki.coronawarnapp.ui.release.NewReleaseInfoFragment"
-        android:label="NewReleaseInfoFragment">
+        android:name="de.rki.coronawarnapp.release.NewReleaseInfoFragment"
+        android:label="NewReleaseInfoFragment" >
```

Comment:
Please use tools:layout. it gives an overview when others are looking into the graph later